packages/web/src/utils/Moods.tsx

```diff
@@ -2,119 +2,119 @@ import { Mood } from '@audius/sdk'
 
 export const moodMap: Record<Mood, JSX.Element> = {
   Peaceful: (
-    <div>
+    <span>
```

Comment:
Could changing these break other UI? Or are they only used in InfoLabel

Reply:
just double checked everywhere, we're good!